content/publishing-yaml/distribution.md

```diff
@@ -227,3 +225,17 @@ And then export the filepath on the gradlew task
         cd android && ./gradlew appDistributionUploadRelease
 
 {{</notebox>}}
+
+### Publishing web applications to Firebase Hosting
+
+Publishing web applications to Firebase Hosting With Codemagic publishing to Firebase Hosting is a straight-forward process as the Firebase CLI is already pre-installed on our virtual machines. To get started, you will need to obtain your Firebase token. In order to do that, run `firebase login:ci` in your local terminal.
+
+After running the command, your default browser should prompt for authorization to your Firebase project - when access is granted, the necessary token will appear in your terminal. 
+
+The next step is to add your token to Codemagic. To properly set everything up, please [encrypt](https://github.com/codemagic-ci-cd/codemagic-docs/blob/master/content/building/encrypting) the token using Codemagic UI and set it under your environment variables with the name `FIREBASE_TOKEN`. Now all that is needed to publish to Firebase Hosting is to add the publishing step to your .yaml file
```

Comment:
2. Add your token to Codemagic. ...
...
3. Add the publishing step to your YAML file after the build step.